drv/stm32h7-spi-server/src/main.rs

```diff
@@ -345,7 +345,7 @@ impl ServerImpl {
         // The BufReader/Writer types manage position tracking for us.
 
         // Wrap a buffer reader/writer onto whichever borrows actually exist.
-        const BUFSIZ: usize = 32;
+        const BUFSIZ: usize = 16;
```

Comment:
Why did this change?  A comment on how this value is chosen would be helpful, although maybe it's obvious if I knew more about `LeaseBufReader`'s behavior.

Reply:
Honestly I was just messing around, neither value shows significantly improved performance at this point so I went for the "use less stack" option. I can add a comment.

Reply:
I have added a comment!